tests/save-artifact-legacy.earth

```diff
@@ -8,13 +8,13 @@ savea:
     SAVE ARTIFACT a AS LOCAL a
 
 # second test: saveb is triggered by
-# a BUILD in savec, and savec is inherrited by
+# a BUILD in savec, and savec is inherited by
 # saved. when +saved is run, we should
 # produce files b, c, and d.
 
 saveb:
     FROM alpine:latest
-    RUN echo bannana > b
+    RUN echo banana > b
```

Comment:
I can't tell why this fruit was spelled this way.

`cherry` is spelled correctly.

I haven't fixed all spellings of `banana`...